src/mavlink-router/mainloop.cpp

```diff
@@ -683,7 +683,7 @@ void Mainloop::set_timeout(Timeout *t, uint32_t timeout_msec)
 
 void Mainloop::del_timeout(Timeout *t)
 {
-    if (t) {
+    if (_timeouts && t) {
```

Comment:
maybe that fix should go first in sequence? I think the other code will crash without this